src/emails/UserCreatedNotification.tsx

```diff
@@ -0,0 +1,37 @@
+import EmailContent from "types/EmailContent"
+import UserDetails from "types/UserDetails"
+import { UserGroupResult } from "types/UserGroup"
+
+interface Props {
+  user: UserDetails
+}
+
+function printGroups(groups: UserGroupResult[]): string {
+  let groupString = ""
+
+  groups.forEach((group) => {
+    groupString += `${group.friendly_name}, `
+  })
+
+  const removeTrailingCommasAndSpaces = /,\s*$/
+
+  return groupString.replace(removeTrailingCommasAndSpaces, "")
+}
+
+const UserCreatedNotificationText = ({ user }: Props): string =>
+  `This is an automated message notifying you of the following Bichard account creation.
+  
+            First name: ${user.forenames}
+             Last name: ${user.surname}
+                 Email: ${user.emailAddress}
+              Endorser: ${user.endorsedBy}
+    Permissions Groups: ${printGroups(user.groups)}
```

Comment:
was username required as well? or was email address enough for them to figure things out?

Reply:
username was not in the ticket, makes sense as the purpose of this email is to let someone know a person has created an account.